Automattic/wp-calypso
# Updating copy and svg icon for video celebration modal.

#### Proposed Changes

* Copy and image update for the VideoPress onboarding celebration modal:

<img width="689" alt="Screenshot 2022-12-02 at 2 32 56 PM" src="https://user-images.githubusercontent.com/789137/205394869-4a3acce0-bb7c-4dec-8e66-77403659828e.png">

#### Testing Instructions



1. run `install-plugin.sh editing-toolkit update/videopress-onboarding-celebration-modal` on your wpcom sandbox
2. go through the onboarding flow and sandbox the new domain before you get to the editor. Upload your first video and save the page
3. you should see the new celebration modal with play button graphic and updated copy.

#### Pre-merge Checklist





Related to #
